core/vote/vote_manager.go

```diff
@@ -139,6 +141,15 @@ func (voteManager *VoteManager) loop() {
 			}
 
 			curHead := cHead.Block.Header()
+			if p, ok := voteManager.engine.(*parlia.Parlia); ok {
+				nextBlockMinedTime := time.Unix(int64((curHead.Time + p.Period())), 0)
+				timeForBroadcast := 100 * time.Millisecond
+				if time.Now().Add(timeForBroadcast).After(nextBlockMinedTime) {
+					log.Debug("too late to vote", "Head.Time(Second)", curHead.Time, "Now(Millisecond)", time.Now().UnixMilli())
```

Comment:
I think we can change the log level to Info

Reply:
> I think we can change the log level to Warning

accepted